datagateway_api/src/datagateway_api/icat/filters.py

```diff
@@ -210,13 +210,20 @@ def apply_filter(self, query):
 
 
 class PythonICATSkipFilter(SkipFilter):
-    def __init__(self, skip_value):
+    def __init__(self, skip_value, client_use="datagateway_api"):
```

Comment:
Perhaps another name for `client_use` might be more appropriate? `filter_use` or an alternative you can think of